### DISCUSSION TITLE
Question: How important is it that source images are divisble by 8 pixels in both height and width?

### DISCUSSION THREAD

**Discussion by marcsyp** [lllyasviel/ControlNet#253] [General]
I know some image sampling processes are most efficient in buckets of 8x8 pixels, so I'm wondering if providing an image to controlnet that is a few pixels off and has to be resampled to an 8-factor resolution has a significant impact on the result quality.  Any thoughts?

Thanks.

**Comment by geroldmeisinger** [comment]
https://github.com/lllyasviel/ControlNet/issues/224#issuecomment-1454997557: "SD can be trained with any resolution as long as the number can be mod by 64." Also see https://github.com/lllyasviel/ControlNet/issues/365
